docs/decisions/0047-ocr-engine-selection.md

```diff
@@ -0,0 +1,158 @@
+---
+parent: Decision Records
+nav_order: 47
+---
+# OCR Engine Selection for JabRef
+
+## Context and Problem Statement
+
+JabRef requires an OCR engine to extract text from scanned PDFs and image-based academic documents. Tesseract is currently implemented, but accuracy varies significantly with document quality and type. Academic documents present unique challenges: mathematical notation, multiple languages, complex layouts, tables, and mixed handwritten/printed content. Which OCR engine(s) should JabRef adopt to best serve its academic user base while balancing accuracy, cost, privacy, and implementation complexity?
+
+## Decision Drivers
+
+* Accuracy on academic document types (printed papers, scanned books, handwritten notes)
```

Comment:
Please provide example documents.  Maybe even do your own test. I think, for the start, two documents are enough.

I don't know why "printed papers" is there - shouldn't it also be "scanned papers"?